functions-and-scope.js

```diff
@@ -3,9 +3,23 @@
 // Tip: je mag hier geen ingebouwde object methoden gebruiken, dus daar hoef je niet naar te kijken.
 
 const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
+const class2 = [6, 8, 4, 7, 2];
+
+function gradesAbove(arr) {
+    let counter = 0
+    for (let i = 0; i < arr.length; i++) {
+        if (arr[i] >= 8) {
+            counter = counter + 1
```

Comment:
Je zou dit wat kunnen versimpelen:
```suggestion
            counter++;
```